charts/fybrik/files/taxonomy/fybrik_application.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
This file is a duplicate of the files in taxonomy/objects - would be better to copy the files during a build process rather than have a duplicate copy.

Reply:
Ok I deleted the `config/taxonomy/objects` directory and moved all the files here